src/components/Navigation.js

```diff
@@ -1,35 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './navigation.css';
 import { NavLink } from 'react-router-dom';
+import Hamburger from 'hamburger-react';
 import logo from '../images/icons/logo.png';
 
-const Navigation = () => (
-  <div>
-    <nav className="navigation">
-      <section className="logo-div">
-        <img className="logo-image" src={logo} alt="Maths Magician Logo" />
-        Maths Magician
-      </section>
-      <section>
-        <ul className="nav-ul">
-          <li>
-            <NavLink className="nav-link" to="/">
-              Home
-            </NavLink>
-          </li>
-          <li>
-            <NavLink className="nav-link" to="/calculator">
-              Calculator
-            </NavLink>
-          </li>
-          <li>
-            <NavLink className="nav-link" to="/quotes">
-              Quotes
-            </NavLink>
-          </li>
-        </ul>
-      </section>
-    </nav>
-  </div>
-);
+function Navigation() {
+  const [isOpen, isClosed] = useState(false);
```

Comment:
As you already know the `useState` Hook returns an array, which has a state on its first index and a setter function on the second index.
to apply React best practices the setter function name have to be started with the `set` Key word and the name has to be in `camelCase` format.